New package: JLD2Lz4 v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: JLD2Lz4
- Repository: https://github.com/JuliaIO/JLD2.jl
- Created by: @JonasIsensee
- Version: v0.1.0
- Commit: 0fb0caccd5eb2b10d70221c703cdaec5512fdae1
- Git reference: HEAD
- Description: HDF5-compatible file format in pure Julia
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
Addon package for JLD2 that provides the new `Lz4Filter()` for dataset compression using the Lz4 codec.

This is split off from base JLD2 to not always require the binary dependency of the codec.
`````
<!-- END RELEASE NOTES -->

JonasIsensee commented:
I was a bit too quick with registration.

The corresponding new JLD2 version was not yet available to run the automerge test.

[no block]

JuliaHubRegistrator committed:
New package: JLD2Lz4 v0.1.0

UUID: 49ca2578-dfa4-4b24-b5ab-b610893dc52a
Repo: https://github.com/JuliaIO/JLD2.jl.git
Tree: 7477bf0b352f9c9d8990e098f631b9d17c8be948

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
